_data/index.yml

```diff
@@ -22,6 +22,15 @@ apps:
       - path: crossref-ojs-manual
       - path: gdpr
       - path: translation
+- path: disc-sust
```

Comment:
@rcgillis When adding a new card, only this line is needed. This should be named based on a card which would be created in `_includes/cards/ojs3`. It should also be indented to align with the list above it. Other elements below this (lines 26-33) are not needed for this file as they are found in the card file.

Reply:
See `_includes/cards/ojs3/crossref-ojs-manual.md` for an example of what the cards should look like.